app/views/comments/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<h2 class='headline'>Add Comment</h2>
+
+<% flash.each do |type, msg| %>
+  <div>
+    <%= msg %>
+  </div>
+<% end %>
+
+	<%= form_with model: @comment, url: user_post_comments_path, scope: :comment, class: 'the_form' do |f| %>
```

Comment:
There is an error when the user tries to create a new comment and also when he tries to like the post. Kindly see the screenshot below
![comment_error](https://user-images.githubusercontent.com/35504149/167018749-5251e305-60d7-4fe3-b05f-50ae771f7010.png)
.

Reply:
@SajjadAhmad14 I've checked and it's working on my side, kindly can you check again.